Amazon.Ion.ObjectMapper/IonObjectSerializer.cs

```diff
@@ -105,6 +111,20 @@ public void Serialize(IIonWriter writer, object item)
                 writer.SetFieldName(GetFieldName(field));
                 ionSerializer.Serialize(writer, fieldValue);
             }
+
+            foreach (var method in targetType.GetMethods())
```

Comment:
Can we add a comment to capture the intent of the loop and the continue condition.

Reply:
Done.